pkg/analyzer/analyzer.go

```diff
@@ -102,17 +102,21 @@ func run(pass *analysis.Pass) (interface{}, error) {
 }
 
 func reportAboutErrorType(pass *analysis.Pass, typePos token.Pos, typeName string, isArrayType bool) {
-	var form string
+	var form strings.Builder
```

Comment:
1) It became inconsistent with `reportAboutErrorVar`.

2) It's small constant probably optimised by compiler. Now code more complex without without objective reasons for it.

3) Profiles show that it only got worse, which is expected:
```
$ go tool pprof basic.mem
top 10
Showing nodes accounting for 2347.30kB, 100% of 2347.30kB total
Showing top 10 nodes out of 27
      flat  flat%   sum%        cum   cum%
 1025.12kB 43.67% 43.67%  1025.12kB 43.67%  runtime.allocm
  809.97kB 34.51% 78.18%   809.97kB 34.51%  golang.org/x/tools/go/internal/gcimporter.(*fakeFileSet).setLines.func1
  512.20kB 21.82%   100%   512.20kB 21.82%  runtime.malg
...
```

```
$ go tool pprof bldr.mem
top 10
Showing nodes accounting for 3886.26kB, 100% of 3886.26kB total
Showing top 10 nodes out of 39
      flat  flat%   sum%        cum   cum%
 2050.25kB 52.76% 52.76%  2050.25kB 52.76%  runtime.allocm
  809.97kB 20.84% 73.60%   809.97kB 20.84%  golang.org/x/tools/go/internal/gcimporter.(*fakeFileSet).setLines.func1
     514kB 13.23% 86.82%      514kB 13.23%  sync.(*Map).LoadOrStore
  512.04kB 13.18%   100%   512.04kB 13.18%  runtime.bgscavenge
...
```